src/library/impl/meta_reader/table.h

```diff
@@ -129,6 +157,11 @@ namespace xlang::meta::reader
         {
             return m_row_count < (1 << 16) ? 2 : 4;
         }
+
+        static constexpr bool is_little_endian_processor()
```

Comment:
reinterpret_cast (and its moral equivalents) are forbidden in constexpr functions. clang correctly rejects this code.

Reply:
I've pushed a change that replaces this code with something that will hold us over until we get C++20's std::endian.